utils.js

```diff
@@ -0,0 +1,25 @@
+const utils = require('@percy/sdk-utils');
+const { Cache } = require('./cache');
+
+class Utils {
+  static projectType() {
+    return utils.percy?.type;
+  }
+
+  static async captureAutomateScreenshot(data) {
+    return await utils.captureAutomateScreenshot(data);
+  }
+
+  static async getSessionId(page) {
+    /* It is browser's guid maintained by playwright, considering it is unique for one automate session
+     will use it to cache the session details */
+    const browserId = page._parent._parent._guid;
+    return await Cache.withCache(Cache.sessionId, browserId, async () => {
+      return JSON.parse(await page.evaluate(/* istanbul ignore next */ _ => { }, `browserstack_executor: ${JSON.stringify({ action: 'getSessionDetails' })}`));
```

Comment:
1. Do we need getSessionDetail, to get keyObjects in seleniumHub side, to send percyScreenshotCommand ?
2. The problem can be this will show a log on the user's dashboard that getSessionDetails is been called everytime the older Cache is cleared. If it's been already discussed then we can ignore this.